src/utils/analytics.test.js

```diff
@@ -117,8 +106,8 @@ describe('Analytics Util', () => {
   });
 
   it('Should call FlashMessageHolder.addMessage 7 days after it was closed', () => {
-    const wrapper = mount(<FlashMessageHolder />, options);
-    const dialogWrapper = mount(<DialogHolder {...props} />, options);
+    const wrapper = mount(<FlashMessageHolder />);
+    const dialogWrapper = mount(<DialogHolder {...props} />);
```

Comment:
In this case will be useful too, you can have 2 setup functions for these wrapper, setupFlashMessage and setupDialogHolder